docs/src/getting_started.md

```diff
@@ -211,9 +211,6 @@ reset_rme()  # Reset RME to clear any previous runs
 # Note: if the Julia runtime crashes, check that the specified data file location is correct
 @RME runCreate(name::Cstring, start_year::Cint, end_year::Cint, RCP_scen::Cstring, gcm::Cstring, reps::Cint)::Cint
 
-# Add 3 DHW enhancement to outplanted corals
-set_option("restoration_dhw_tolerance_outplants", 3)
-
```

Comment:
Could you clarify why the outplanting removed in the example?

The commit message for this change just says "fix docs"?

Reply:
Are we dropping backward compatibility?